### PR TITLE
join users and category tables to /post request

### PR DESCRIPTION
**Pull request by nicholasvaldez**
# Description 

Joined users and categories tables to /posts response when client requests all posts so that client can see the author's first and last name, and the category of a post.
Altered users.py in models, to initialize keys to empty strings. 

# Steps to Test
1. Fetch nick-all-posts and switch to it
2. Start server in debugger mode
3. Open postman and make a request to /posts
4. Verify the response contains both Users and Categories embedded 
5. Verify in Users key, only First_name and last_name keys have values, and the rest empty strings

@jesmeeker @MaeYoungPhan @lauren-hanson @JLinkous15

### CODE REVIEW

**Comment by jesmeeker** [comment]
Does this close a specific ticket? Wanted to make sure the ticket was fulfilled if so.

File: views/repository.py [hunk 1]
```diff
@@ -75,8 +76,15 @@ def all(resource):
...
-                p.approved
+                p.approved,
+                u.first_name first_name,
+                u.last_name last_name,
+                c.label label
             FROM posts p
+            JOIN Users u
+                ON u.id = p.user_id
+            JOIN Categories c 
+                ON c.id = p.category_id
```
Comment (jesmeeker) [inline views/repository.py:87]: Add sql query to ORDER BY

**Review by jesmeeker — CHANGES_REQUESTED** [review]

**Review by jesmeeker — APPROVED** [review]

**Comment by jesmeeker** [comment]
Ready for merge!

File: views/repository.py (hunk 1 above)
Comment (nicholasvaldez) [reply views/repository.py:87]: 🦖 
👍

**Pull request merged**